design-documents/graph-ql/coverage/cart/Cart.graphqls

```diff
@@ -3,15 +3,16 @@ type Query {
 }
 
 input CartQueryInput {
-    cart_id: String!
+    cart_id: ID!
```

Comment:
We actually don't have `CartQueryInput` type, will create a separate PR to update.